src/openforms/submissions/tests/test_resume_form_view.py

```diff
@@ -50,11 +50,13 @@ def test_good_token_and_submission_redirect_and_add_submission_to_session(self):
         response = self.client.get(endpoint)
 
         f = furl(submission.form_url)
-        # furl adds paths with the /= operator
-        f /= "stap"
-        f /= submission.get_last_completed_step().form_step.slug
-        # Add the submission uuid to the query param
-        f.add({"submission_uuid": submission.uuid})
+        # Add the redirect path and submission uuid to the query param
+        f.add(
+            {
+                "_of_action": "stap",
+                "_of_action_params": f"step_slug={submission.get_last_completed_step().form_step.slug}&submission_uuid={submission.uuid}",
```

Comment:
I strongly dislike these long lines with complicated f-strings that also require you to manually do the URL encoding.

At this point, all I care about is that:

* the right parameters are specified
* they have been serialized (urlencoded or JSON, it's not relevant)

The exact form and order of parameters is not relevant and a source of hard-to-debug tests, so I encourage writing a small helper for this.